code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -534,6 +534,14 @@
 		/obj/item/stock_parts/capacitor = 1)
 	def_components = list(/obj/item/stock_parts/cell = /obj/item/stock_parts/cell/high/empty)
 
+/obj/item/circuitboard/machine/rad_collector
+	name = "Radiation Collector (Machine Board)"
+	build_path = /obj/machinery/power/rad_collector
+	req_components = list(
```

Comment:
very cheap I would suggest it requires a matter bin and reinforced plasma glass too